modules/pom.xml

```diff
@@ -24,6 +24,7 @@
         <module.xml.aliases.basedir>${basedir}/src/main/aliases</module.xml.aliases.basedir>
         <hibernate.ogm.module.xml>${module.xml.basedir}/ogm/core/module.xml</hibernate.ogm.module.xml>
         <hibernate.ogm.module.slot>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}</hibernate.ogm.module.slot>
+        <hibernate.ogm.jipijapa.module.slot>${project.version}</hibernate.ogm.jipijapa.module.slot>
```

Comment:
Yes. The "hibernate ogm module slot" is usually resolved to major.minor ( "5.0" or "5.1", etc..) so that people can refer to that, and we give them the latest within the chosen major.minor.

Some modules are internal though: people don't depend on them directly, but we rather inject the "right" versions for them. In this case we use the full version identifier, as we might be changing details which are not necessarily compatible (i.e. internal stuff).

for example, see the module "hibernate.ogm.neo4j-internal.module.slot" below: it also uses the full project version.